src/zenml/artifacts/utils.py

```diff
@@ -442,6 +445,75 @@ def load_artifact_from_response(artifact: "ArtifactVersionResponse") -> Any:
     )
 
 
+def save_artifact_binary_from_response(
```

Comment:
Just to keep things consistent, I would also rename this function and adjust the docstring accordingly.